src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+
+    subtract(number) {
+      this.result -= number;
+
+      return this;
+    },
+
+    multiply(number) {
+      this.result *= number;
+
+      return this;
+    },
+
+    divide(number) {
+      if (number !== 0) {
+        this.result /= number;
+      }
```

Comment:
[CODE KNOWLEDGE] - The check for division by zero is unnecessary since JavaScript will return Infinity when dividing by zero, which is a valid result for a calculator. You can remove the if condition and simply use the division assignment operator. Example: `this.result /= number;`